newIDE/app/src/Utils/GDevelopServices/Project.js

```diff
@@ -261,7 +261,7 @@ export const clearCloudProjectCredentials = async (): Promise<void> => {
 
 export const createCloudProject = async (
   authenticatedUser: AuthenticatedUser,
-  cloudProjectCreationPayload: {| name: string, gameId?: string |}
+  cloudProjectCreationPayload: {| name: string, gameId: string |}
```

Comment:
thanks to this, we shouldn't created a project without a gameId anymore

Reply:
The typing should indeed enforce this, nice!